libwavformat/input.c

```diff
@@ -445,8 +445,9 @@ t_wav_uint32 waveformat_input_process_int16(t_wav_input_file * p_file,t_wav_int1
 	return samples_read;
 }
 
-void waveformat_input_close(t_wav_input_file * /*p_file*/)
+void waveformat_input_close(t_wav_input_file * p_file)
 {
+	(void)p_file;

```

Comment:
What is the difference between this and unnamed parameters?

Reply:
Unnamed parameters are, [per the CI logs](https://github.com/dragonCodecs/libAudio/actions/runs/6029542457/job/16359327417#step:11:203), a GNU extension and a C23 feature - while this leaves the parameter named but tells the compiler not to warn about it not being used.